mtools/mloginfo/sections/connection_section.py

```diff
@@ -61,6 +93,38 @@ def run(self):
                     ip, _ = tokens[2].split(':')
                 ip_closed[ip] += 1
 
+                if genstats:
+                    dt = logevent.datetime
+
+                    #The connection id value is stored just before end connection -> [conn385] end connection
+                    tokens_conn = line[:pos].split(' ')
+                    end_connid = tokens_conn[3].strip('[|conn|]')
+
+                    #Check if we have the starting of this connid
```

Comment:
Think this needs some check of the validity of `end_connid` before trying to use as a dict key

Reply:
done